system_modules/napaudio/src/audio/core/audionodemanager.cpp

```diff
@@ -8,96 +8,121 @@
 #include <nap/logger.h>
 #include <nap/core.h>
 
+#ifndef RASPBERRY
+    #include <xmmintrin.h>
+#endif
+
 namespace nap
 {
 	namespace audio
 	{
-		
+
 		NodeManager::~NodeManager()
 		{
 			// Tell the nodes that their node manager is outta here, so they won't try to unregister themselves in their dtors.
-			for (auto& node : mNodes)
+			for (auto& node : mProcesses)
 			{
 				node->mNodeManager = nullptr;
 				node->mRegisteredWithNodeManager.store(false);
 			}
 		}
-		
-		
+
+
 		void NodeManager::process(float** inputBuffer, float** outputBuffer, unsigned long framesPerBuffer)
 		{
+#ifndef RASPBERRY
```

Comment:
don't understand whats happening here, maybe add a bit more context or information about why this is necessary ?

Reply:
Its an optimization, denormals in floats are disabled on the audio thread. This is not supported on ARM.

Reply:
I added a better explanation in the comments